### PR TITLE
Add Guide for OpenTelemetry

### LINKED ISSUE #16093: Add Quickstart/Guide for OpenTelemetry extension
## Description

Add a guide and quickstart for OpenTelemetry

### PR DESCRIPTION
**Pull request by kenfinnigan**
Resolves #16093

### CODE REVIEW

**Comment by kenfinnigan** [comment]
Merge https://github.com/quarkusio/quarkus-quickstarts/pull/872 when approved/merged

File: docs/src/main/asciidoc/opentelemetry.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus - Using OpenTelemetry
+
+include::./attributes.adoc[]
+
+This guide explains how your Quarkus application can utilize OpenTelemetry to provide distributed tracing for
```
Comment (radcortez) [inline docs/src/main/asciidoc/opentelemetry.adoc:10]: Add an external link to the OT project.

File: docs/src/main/asciidoc/opentelemetry.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+quarkus.opentelemetry.tracer.exporter.jaeger.endpoint=http://localhost:14250 // <3>
+----
+
+<1> All spans created from the application will include an OpenTelemetry `Resource` indicating the span was created by the `myservice` application. If not set, it will default to the artifact id.
+<2> Whether OpenTelemetry is enabled or not. The default is `true`, but shown here to indicate how it can be disabled
+<3> gRPC endpoint for sending spans.
+
+== Run the application
+
+The first step is to start the tracing system to collect and display the captured traces:
```
Comment (radcortez) [inline docs/src/main/asciidoc/opentelemetry.adoc:125]: Add that we will be use Jaeger for this and a link to Jaeger.

**Review by radcortez — APPROVED** [review]

**Comment by radcortez** [comment]
Left a couple of suggestions. Please check if they make sense.

**Comment by kenfinnigan** [comment]
How's that @radcortez?

**Review by gsmet — APPROVED** [review]

**Review by radcortez — APPROVED** [review]

**Pull request merged**